src/App.js

```diff
@@ -1,24 +1,50 @@
-import logo from './logo.svg';
-import './App.css';
+/* eslint-disable import/no-named-as-default-member */
+import React, { useEffect, lazy, Suspense } from 'react';
+import { useDispatch } from 'react-redux';
+import { Route, Routes } from 'react-router-dom';
+import CryptoJS from 'crypto-js';
+import { addUSER } from './redux/user/user';
+import Loader from './components/Loader';
+import Nav from './components/Nav';
+import DeleteTours from './components/DeleteTours';
+import TourDetails from './components/TourDetails';
+import CreateTour from './components/CreateTour';
+import Reservation from './components/Reservation';
+import SignUpPage from './components/SignUpPage';
+import LogInPage from './components/LogInPage';
+import ShowReservation from './components/ShowReservation';
+import './styles/index.css';
+
+const MainPage = lazy(() => import('./components/MainPage'));
 
 function App() {
```

Comment:
- [ ] Upon running the app, this happens (blank screen) (tested in Chrome):

![image](https://user-images.githubusercontent.com/57421823/199584163-95d6d787-93cd-40fe-8646-fe6ea283835a.png)

Please, correct this bug so your app works smoothly as intended, and can be properly reviewed. 💪 

- [ ] You are using the "function" keyword for this functional component.  Please, kindly refactor it to ES6 arrow function.

The "function" keyword has been soft deprecated in almost all of the software industry.  If you refactor to ES6 arrow function, your app will comply with the latest standards.  💪